setup.cfg

```diff
@@ -0,0 +1,3 @@
+[flake8]
+exclude = node_modules, docs/components/collapse.py
```

Comment:
Why are we excluding `collapse.py`?

Reply:
Basically, collapse is not a valid Python file -- it depends on an undefined variable app.

When we build the documentation, we load the code with `exec` and an overriden set of local variables in which we inject `app`.

The complexity around the component pages is that they:
- need to work and actually render the right information
- be somewhat intuitive for the user to understand when they read the code.

Reply:
https://dash-bootstrap-components.opensource.asidatascience.com/l/components/collapse

Reply:
That makes sense, thanks for the clarification!